Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN echo "force-unsafe-io" > /etc/dpkg/dpkg.cfg.d/02apt-speedup \
 
 # Install prerequisites
 RUN set -x && \
-  apt-get update && apt-get -y upgrade && \
```

Comment:
you miss what I meant, I mean what command are you starting the container with?  I run mine with 
```
docker run -d --restart=always --name xeoma --net=host -v /etc/localtime:/etc/localtime:ro -v /share/docker/xeoma:/usr/local/Xeoma -v /mnt/archive:/usr/local/Xeoma/XeomaArchive jedimonkey/xeoma:beta
```